gulpfile.js

```diff
@@ -72,9 +72,11 @@ const config = {
 
     function doSCSS (component) {
         return gulp.src(component.src)
-            .pipe(scssLint({
+            // https://www.npmjs.com/package/gulp-scss-lint#install
```

Comment:
I wouldn't commit this change, linting should still be done, but can be disabled if desired...